cli/src/commands/exception.rs

```diff
@@ -0,0 +1,361 @@
+    })];
+
+    match project {
+        Some(project_name) => {
+            let group = group.map(String::as_str);
+            let project_id = api.get_project_id(project_name, org, group).await?.to_string();
+            api.project_suppress(&project_id, &suppressions).await?;
+        },
+        None => api.group_suppress(org, group.unwrap(), &suppressions).await?,
+    }
+
+    print_user_success!("Successfully added suppression for {}", purl.to_string());
+
+    Ok(ExitCode::Ok)
+}
+
+/// Handle `phylum exception remove` subcommand.
+pub async fn handle_remove(api: &PhylumApi, matches: &ArgMatches, config: Config) -> CommandResult {
+    let ecosystem = matches.get_one::<String>("ecosystem");
+    let project = matches.get_one::<String>("project");
+    let version = matches.get_one::<String>("version");
+    let group = matches.get_one::<String>("group");
+    let name = matches.get_one::<String>("name");
+    let purl = matches.get_one::<String>("purl");
+    let tag = matches.get_one::<String>("tag");
+    let id = matches.get_one::<String>("id");
+    let org = config.org();
+
+    let mut exceptions = match matches.get_one::<String>("project") {
+        Some(project_name) => {
+            let group = group.map(String::as_str);
+            let project_id = api.get_project_id(project_name, org, group).await?.to_string();
+            api.project_preferences(&project_id).await?
+        },
+        None => api.group_preferences(config.org(), group.unwrap()).await?,
+    };
+
+    // Filter issue suppressions with CLI args.
+    if tag.is_some() || id.is_some() {
+        exceptions.ignored_issues.retain(|issue| {
+            id.is_none_or(|id| id == &issue.id) && tag.is_none_or(|tag| tag == &issue.tag)
+        });
+    }
+
+    // Filter package suppressions with CLI args.
+    if ecosystem.is_some() || name.is_some() || version.is_some() || purl.is_some() {
+        let purl = purl.map(|purl| Purl::from_str(purl));
+        let (ecosystem, namespace, name, version) = match purl {
+            Some(Ok(ref purl)) => {
+                let package_type = Cow::Owned(purl.package_type().to_string());
+                (Some(package_type), purl.namespace(), Some(purl.name()), purl.version())
+            },
+            Some(Err(err)) => return Err(err.into()),
+            None => (
+                ecosystem.map(Cow::Borrowed),
+                None,
+                name.map(String::as_str),
+                version.map(String::as_str),
+            ),
+        };
+
+        exceptions.ignored_packages.retain(|pkg| {
+            let purl = match Purl::from_str(&pkg.purl) {
+                Ok(purl) => purl,
+                Err(_) => return false,
+            };
+
+            let package_type = purl.package_type().to_string();
+            ecosystem.as_ref().is_none_or(|ecosystem| **ecosystem == package_type)
+                && namespace.is_none_or(|namespace| Some(namespace) == purl.namespace())
```

Comment:
It doesn't unsuppress anything directly, it just suggests options. I could special-case the purl here, but I didn't because it would unnecessarily complicate things. It seems fine to suggest ones with and without namespace?